commit.go

```diff
@@ -77,12 +82,31 @@ func (pi *PersonInfo) resolve(p []string) (interface{}, []string, error) {
 	case "email":
 		return pi.Email, p[1:], nil
 	case "date":
-		return pi.Date + " " + pi.Timezone, p[1:], nil
+		date, err := pi.date()
+		if err != nil {
+			return nil, nil, err
+		}
+		return date.Format(time.RFC3339), p[1:], nil
 	default:
 		return nil, nil, errors.New("no such link")
 	}
 }
 
+func (pi *PersonInfo) date() (*time.Time, error) {
+	sec, err := strconv.ParseInt(pi.Date, 10, 64)
+	if err != nil {
+		return nil, err
+	}
+	zoneOffset, err := strconv.Atoi(pi.Timezone)
+	if err != nil {
+		return nil, err
+	}
+	hr, mm := zoneOffset/100, zoneOffset%100
```

Comment:
Actually, will this work for negative offsets - `zoneOffset%100`?

Reply:
(not that there are many places with non-1h offsets, but they exist - https://en.wikipedia.org/wiki/List_of_UTC_time_offsets)

Reply:
Not sure, but I can add tests for these cases.